website/content/commands/operator/raft.mdx

```diff
@@ -104,3 +104,29 @@ Usage: `consul operator raft remove-peer -address="IP:port"`
 - `-id` - ID of the server to remove.
 
 The return code will indicate success or failure.
+
+## transfer-leader
+
+Corresponding HTTP API Endpoint: [\[POST\] /v1/operator/raft/transfer-leader](/api-docs/operator/raft#transfer-leader)
+
+This command transfer the Consul leader to another server. If an `id` is provided, raft leadership will be transferred
```

Comment:
Is it worthwhile to mention _why_ someone might want to use this command?

For example:
"This command provides a way to transition leadership without restarting the leader node, maintaining quorum and workload capacity."